### DISCUSSION TITLE
Is Mask-RCNN a fully convolutional architecture?

### DISCUSSION THREAD

**Discussion by justlike-prog** [facebookresearch/detectron2#4206] [Q&A]
Is Mask-RCNN a fully convolutional architecture (from what I see yes, because there are no dense layers but just wanted to make sure)? Thus can I feed differently sized images for inference without resizing? Also while training could I feed the original sized images without resizing?

**Comment by ppwwyyxx** [comment]
> Thus can I feed differently sized images for inference without resizing

you can.

> Also while training could I feed the original sized images without resizing?

you can, though this is not common
